doc/whats_new/v1.4.rst

```diff
@@ -167,6 +167,13 @@ Changelog
   :pr:`26315` and :pr:`27098` by :user:`Mateusz Sokół <mtsokol>`,
   :user:`Olivier Grisel <ogrisel>` and :user:`Edoardo Abati <EdAbati>`.
 
+- |Feature| :class:`decomposition.PCA` now supports :class:`scipy.sparse.sparray`
+  and :class:`scipy.sparse.spmatrix` inputs when using the `arpack` solver.
+  When used on sparse data like :func:`datasets.fetch_20newsgroups_vectorized` this
+  can lead to speed ups of 100x (single threaded) and 70x lower memory usage.
+  Based on :user:`Alexander Tarashansky <atarashansky>`'s implementation in `scanpy <https://github.com/scverse/scanpy>`.
+  :pr:`18689` by :user:`Isaac Virshup <ivirshup>` and :user:`Andrey Portnoy <andportnoy>`.
```

Comment:
> This is a simple adaptation based on my observation on my machine

I reran this and got those relative speedups + memory usage. The single threaded part is relevant for speed, since dense matrix multiplication can be multithreaded.

Reply:
I see. It might depend on the machine one uses and the version of the implementation of BLAS.

> The single threaded part is relevant for speed, since dense matrix multiplication can be multithreaded.

Could you explain more about what you mean here? Was this a comparison when running implementation using a single thread?

Reply:
> Was this a comparison when running implementation using a single thread?

In my recent rerunning of the benchmark: yes. Well, it was with openblas but using threadpoolctl to get it to use one thread.

I don't remember how I ran the initial benchmark, but it was a while ago.

Reply:
Well in practice most people will run dense PCA with as many threads as cores (typically 8 to 16 nowadays). But fair enough.